src/main/java/com/klodnicki/bike/rest/API/Bike/config/SpringSecurity.java

```diff
@@ -40,9 +41,10 @@ public SecurityFilterChain securityFilterChain(HttpSecurity httpSecurity) throws
         .requestMatchers(HttpMethod.GET, "/api/info").permitAll()
         .requestMatchers("/api/login/**").hasAnyRole("ADMIN", "USER")
         .anyRequest()
-        .authenticated())
+        .permitAll())
```

Comment:
I do not agree that permitAll for any request is okay here. It's security breach with huge impact.

Reply:
I got it! and I change it into authenticated!